test/filter_stream_test.cpp

```diff
@@ -0,0 +1,178 @@
+#include <unifex/filter_stream.hpp>
+
+#include <unifex/for_each.hpp>
+#include <unifex/just_void_or_done.hpp>
+#include <unifex/range_stream.hpp>
+#include <unifex/reduce_stream.hpp>
+#include <unifex/sync_wait.hpp>
+#include <unifex/task.hpp>
```

Comment:
I think you're not using this, and it's causing build errors in the CI configurations that don't support coroutines.
```suggestion
```